README.md

```diff
@@ -1,3 +1,43 @@
+# Project Summary
+
+In this project I have built an API that allows you to access and interact with a database to provide information to front-end architecture. The purpose of this was to mimic real world backend services and gain experience in that field.
+
+Here is a link to the hosted version of this API:
+[NC-Games-Hosted](https://github.com/DarthShan/NC-Games-Hosted)
```

Comment:
Is this meant to be linking to this repo?  If you are saying it is hosted it should be a link to where you have hosted it (on cyclic)

Reply:
Yeah I've kinda messed it up I just realised haha. I created the README on the non hosted file but then when I pushed it pushed it to the hosted one and I think it's something to do with making the server yesterday